eslint.config.mjs

```diff
@@ -0,0 +1,74 @@
+import { defineConfig, globalIgnores } from 'eslint/config';
+import importPlugin from 'eslint-plugin-import';
+import studio from 'eslint-plugin-studio';
+import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
+import globals from 'globals';
+import tsEslint from 'typescript-eslint';
+import js from '@eslint/js';
+import reactHooks from 'eslint-plugin-react-hooks';
+
+export default defineConfig(
+	globalIgnores( [ '**/node_modules/', '**/dist/', '**/out/', '**/wp-files/', '**/vendor/' ] ),
+	js.configs.recommended,
+	tsEslint.configs.recommended,
+	importPlugin.flatConfigs.recommended,
+	importPlugin.flatConfigs.electron,
+	importPlugin.flatConfigs.typescript,
+	eslintPluginPrettierRecommended,
+	reactHooks.configs.flat.recommended,
+	{
+		plugins: {
+			studio: studio,
+		},
+		languageOptions: {
+			globals: {
+				...globals.browser,
+				...globals.node,
+			},
+			ecmaVersion: 5,
```

Comment:
Nit: I don't think we need to support `ecmaVersion: 5`, so we could remove this and use the default "latest" as [recommended in the documentation](https://eslint.org/docs/latest/use/configure/language-options#specifying-javascript-options)

Reply:
I believe you're right. Vite should handle any transpiling needs with esbuild.